src/irmin-pack/unix/file_manager.ml

```diff
@@ -120,6 +120,10 @@ struct
     if Dict.empty_buffer t.dict then Ok ()
     else
       let* () = Dict.flush t.dict in
+      (* update stats; FIXME we increment on successful flush; elsewhere we increment just
+         before the function call - i.e. we increment whether successful or not; perhaps
+         the following line should come before the call to [Dict.flush]*)
+      Stats.Fm.incr_dict_flushes ();
```

Comment:
where I looked, we update the stats after the function call, but I think its fine either way. Maybe just keep the same convention you pick everywhere else in this file.